teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,201 +4,186 @@
    "cell_type": "markdown",
```

Comment:
<p>This module can't import because of the current python path.</p><p>There is an idea on my computer:</p><pre class="ql-syntax" spellcheck="false">import sys
sys.path.append('../../..')
from teachopencadd.utils import show_pdf
</pre><p>Maybe you have a better idea.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Resolved by updating environment.